modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -40,6 +40,27 @@ It is not returned by default in either command.
 
 The column is a `STRING` value specifying a replacement function/procedure if the used function/procedure is deprecated. Otherwise, it returns `null`.
 
+|===
+[[cypher-deprecations-additions-removals-5.23]]
+== Neo4j 5.23
+
+=== Updated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+RETURN datetime.statement() IS :: TIMESTAMP WITH TIME ZONE
+----
+a|
+Added an alias to the timezone keyword; `TIME ZONE`. 
```

Comment:
is `TIMEZONE` the alias, or `TIME ZONE`?
which one is the default? because in index.adoc above, it looks like `TIME ZONE` is the one we want to use in our examples, which would make `TIMEZONE` the alternative ( = alias ?)

Reply:
Well technically TIMEZONE and TIME ZONE are alias' to other defaults 😅 e.g LOCAL DATETIME is preferred over TIMESTAMP WITHOUT TIMEZONE

Reply:
What should I say instead? :)

Reply:
Maybe:
 "Introduced new GQL conformant aliases to duration types: `TIMESTAMP WITHOUT TIME ZONE` (alias to `LOCAL DATETIME`), `TIME WITHOUT TIME ZONE` (alias to `LOCAL TIME`), `TIMESTAMP WITH TIME ZONE` (alias to `ZONED DATETIME`), and `TIME WITH TIME ZONE` (alias to `ZONED TIME`)."